backend/src/v2/cmd/driver/main.go

```diff
@@ -41,18 +41,18 @@ const (
 
 var (
 	// inputs
-	driverType           = flag.String(driverTypeArg, "", "task driver type, one of ROOT_DAG, DAG, CONTAINER")
-	pipelineName         = flag.String("pipeline_name", "", "pipeline context name")
-	runID                = flag.String("run_id", "", "pipeline run uid")
-	componentSpecJson    = flag.String("component", "{}", "component spec")
-	taskSpecJson         = flag.String("task", "", "task spec")
-	runtimeConfigJson    = flag.String("runtime_config", "", "jobruntime config")
-	iterationIndex       = flag.Int("iteration_index", -1, "iteration index, -1 means not an interation")
-	kubernetesConfigJson = flag.String("kubernetes_config", "", "Kuberntes specific configurations")
+	driverType        = flag.String(driverTypeArg, "", "task driver type, one of ROOT_DAG, DAG, CONTAINER")
+	pipelineName      = flag.String("pipeline_name", "", "pipeline context name")
+	runID             = flag.String("run_id", "", "pipeline run uid")
+	componentSpecJson = flag.String("component", "{}", "component spec")
+	taskSpecJson      = flag.String("task", "", "task spec")
+	runtimeConfigJson = flag.String("runtime_config", "", "jobruntime config")
+	iterationIndex    = flag.Int("iteration_index", -1, "iteration index, -1 means not an interation")
 
 	// container inputs
 	dagExecutionID    = flag.Int64("dag_execution_id", 0, "DAG execution ID")
 	containerSpecJson = flag.String("container", "{}", "container spec")
+	k8sExecConfigJson = flag.String("kubernetes_config", "{}", "kubernetes executor config")
```

Comment:
Moved it here and changed the description as we only pass an executor here, which belongs to container inputs.